bake/bake.go

```diff
@@ -729,6 +745,59 @@ func updateContext(t *build.Inputs, inp *Input) {
 	t.ContextState = &st
 }
 
+// validateContextsEntitlements is a basic check to ensure contexts do not
+// escape local directories when loaded from remote sources. This is to be
+// replaced with proper entitlements support in the future.
+func validateContextsEntitlements(t build.Inputs, inp *Input) error {
+	if inp == nil || inp.State == nil {
+		return nil
+	}
+	if v, ok := os.LookupEnv("BAKE_ALLOW_REMOTE_FS_ACCESS"); ok {
```

Comment:
@crazy-max I think in our github actions we can define this by default as the whole system is in a vm anyway.

Reply:
On GHA they could include `$HOME/.docker` which contains auth token in `config.json` if login step has been done in a previous step.